Cargo.toml

```
@@ -145,9 +144,9 @@ all-features = true
 rustc-args = ["--cfg=docsrs"]
 
 [workspace.dependencies]
-astarte-device-sdk = { path = "./", version = "=0.10.2" }
-astarte-device-sdk-derive = { version = "=0.10.2", path = "./astarte-device-sdk-derive" }
-astarte-interfaces = { path = "./astarte-interfaces", version = "=0.1.1" }
+astarte-device-sdk = { path = "./", version = "=0.10.3" }
+astarte-device-sdk-derive = { version = "=0.10.3", path = "./astarte-device-sdk-derive" }
+astarte-interfaces = "=0.1.1"
```

```suggestion
astarte-interfaces = "0.1.1"
```

We can relax this version, since it will support any patch above 0.1.1